front/src/router/index.js

```diff
@@ -19,7 +19,8 @@ import {
   MyAccommodation,
   MyFood,
   MyTransport,
-  MyLeisure
+  MyLeisure,
+  LoginCard
 } from '@views';
```

Comment:
please, change import one component per line onto several to save vertical space in component